arc-api/server/models/key-store.js

```diff
@@ -0,0 +1,59 @@
+var fs = require('fs-extra');
+var path = require('path');
+var async = require('async');
+
+var homeDir = process.env.HOME || process.env.HOMEPATH ||
+  process.env.USERPROFILE;
+
+/**
+ * Key store
+ * @param options
+ * @constructor
+ */
+function KeyStore(options) {
+  options = options || {};
+  this.root = options.root || path.join(homeDir, '.strongloop');
+  this.perUser = options.perUser || false;
+}
+
+KeyStore.prototype.getStoreFile = function(userId) {
+  var name = 'licenses.json';
```

Comment:
While I agree on principle, the decision to make other modules depend on `~/.strongloop/licenses.json` being the path is somewhat mutually exclusive with making it configurable.

See https://github.com/strongloop/strong-arc/issues/1069#issuecomment-77671719